media-sound/yoshimi/yoshimi-1.7.1-r1.ebuild

```
@@ -35,6 +35,8 @@ CMAKE_USE_DIR="${WORKDIR}/${P}/src"
 
 DOCS=( Changelog README.txt )
 
+PATCHES=( ${FILESDIR}/yoshimi-1.7.1-fix-LV2_Descriptor.patch )
```

I think we need quoting on at least "${FILESDIR}", but I would do the whole thing like:
```suggestion
PATCHES=( "${FILESDIR}/${PN}-1.7.1-fix-LV2_Descriptor.patch"  )
```

Change the ${PN}-1.7.1 to ${P} if this patch is upstreamed.

---

It seems like useless labor, but I push it.